json/nl.json

```diff
@@ -1,46 +1,61 @@
+    "A new verification link has been sent to the email address you provided during registration.": "Er is een nieuwe verificatielink verstuurd naar het e-mailadres dat u ingegeven heeft tijdens uw registratie.",
     "All rights reserved.": "Alle rechten voorbehouden.",
     "Already registered?": "Already registered?",
+    "Are you sure you want to delete your account? Once your account is deleted, all of its resources and data will be permanently deleted. Please enter your password to confirm you would like to permanently delete your account.": "Bent u zeker dat u uw account permanent wilt verwijderen? Als uw account wordt verwijderd, worden alle gekoppelde bestanden en gegevens ook permanent verwijderd. Gelieve uw wachtwoord in te geven om te bevestigen dat u uw account permanent wilt verwijderen.",
     "Before proceeding, please check your email for a verification link.": "Om verder te gaan, check je e-mail voor een verificatielink.",
     "click here to request another": "vraag hier een andere aan",
     "Confirm Password": "Bevestig wachtwoord",
+    "Current Password": "Huidig wachtwoord",
     "Dashboard": "Dashboard",
+    "Delete Account": "Account Verwijderen",
     "E-Mail Address": "E-mailadres",
-    "Email Password Reset Link": "Email Password Reset Link",
+    "Email Password Reset Link": "Verstuur link",
     "Email": "Email",
+    "Ensure your account is using a long, random password to stay secure.": "Zorg ervoor dat uw account een lang, willekeurig wachtwoord gebruikt om veilig te blijven.",
     "Error": "Fout",
     "Forbidden": "Geen toegang",
-    "Forgot your password? No problem. Just let us know your email address and we will email you a password reset link that will allow you to choose a new one.": "Forgot your password? No problem. Just let us know your email address and we will email you a password reset link that will allow you to choose a new one.",
-    "Forgot your password?": "Forgot your password?",
-    "Forgot Your Password?": "Wachtwoord vergeten?",
+    "Forgot your password? No problem. Just let us know your email address and we will email you a password reset link that will allow you to choose a new one.": "Wachtwoord vergeten? Geen probleem. Geef hier uw e-mailadres in en we sturen u een link via mail waarmee u een nieuw wachtwoord kan instellen.",
+    "Forgot your password?": "Wachtwoord vergeten?",
     "Go Home": "Terug naar de voorpagina",
     "Hello!": "Hallo!",
     "hi": "hallo",
+    "If necessary, you may logout of all of your other browser sessions across all of your devices. If you feel your account has been compromised, you should also update your password.": "Indien nodig kunt u zich afmelden op alle andere browser sessies op al uw apparaten. Als u denkt dat uw account is gehackt, kunt u ook best uw wachtwoord aanpassen.",
```

Comment:
This tag was updated yesterday : *"If necessary, you may logout of all of your other browser sessions across all of your devices. Some of your recent sessions are listed below; however, this list may not be exhaustive. If you feel your account has been compromised, you should also update your password"*

Reply:
Ok thanks! I fixed it in the next 2 commits (I'm sorry for the typo).